srml/staking/src/mock.rs

```diff
@@ -16,6 +16,8 @@
 
 //! Test utilities
 
+#![cfg(test)]
```

Comment:
I think we usually put `#[cfg(test)]` in the `src/lib.rs` file above the module declaration.